### PR TITLE
1017 create the Android authentication guide for MA

### PR DESCRIPTION
**Pull request by saudsami**

### CODE REVIEW

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Ok, so, for Android embedded it works in exactly the same way as normal android, right? 

Couple of small comments, plus, the link to the token generator page does not work, you need to add the doc to this product.

File: shared/variables/global.js [hunk 1]
```diff
@@ -110,7 +110,11 @@ export const API_REF_ROOT_CHAT_SDK = `${API_ROOT}/chat-sdk`;
 export const CHAT_SDK_API_ANDROID  = `${API_REF_ROOT_CHAT_SDK}/android/${CHAT_RELEASE_API}`;
 export const CHAT_SDK_API_WEB  = `${API_REF_ROOT_CHAT_SDK}/web/${CHAT_RELEASE_API}`;
 
-
+export const MA = `MA`;
+export const MA_RELEASE_API = `1.8`;
+export const MA_SDK_API_REF_ROOT = `https://docs.agora.io/cn/RTSA/API%20Reference`;
```
[the remark this answers is not in the document]
Comment (saudsami) [reply shared/variables/global.js:115]: Yes, It is very similar to the Video SDK Android implementation.

File: shared/video-sdk/_authentication-workflow.mdx [hunk 2]
```diff
@@ -43,9 +39,10 @@ The following figure shows the call flow you need to implement to create step-up
 
 To follow this procedure the reader is suggested to have:
```
[the remark this answers is not in the document]
Comment (saudsami) [reply shared/video-sdk/_authentication-workflow.mdx:40]: Fixed

File: shared/variables/product.js [hunk 3]
```diff
@@ -56,7 +56,15 @@ const data = {
       PATH: 'server-gateway',
       SDK: 'Server Gateway SDK',
       PRODUCT: 'Server Gateway'
+  },
+
+  'media-accelerator': {
+    NAME: 'Media Accelerator',
+    PATH: 'media-accelerator',
+    SDK: 'RTSA SDK',
```
[the remark this answers is not in the document]
Comment (saudsami) [reply shared/variables/product.js]: fixed

**Pull request merged**